From dee8a903c176fb932c5cc76e136f1537834e198f Mon Sep 17 00:00:00 2001
From: ananjaser1211 <ananjaser@gmail.com>
Date: Tue, 8 Jan 2019 14:35:54 +0000
Subject: [PATCH 3/3] libc: replace undeclared log

---
 libc/bionic/pthread_internal.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libc/bionic/pthread_internal.cpp b/libc/bionic/pthread_internal.cpp
index 4305e2f8e..6c0282009 100644
--- a/libc/bionic/pthread_internal.cpp
+++ b/libc/bionic/pthread_internal.cpp
@@ -116,7 +116,7 @@ pthread_internal_t* __pthread_internal_find(pthread_t thread_id) {
   // Historically we'd return null, but
   if (bionic_get_application_target_sdk_version() >= __ANDROID_API_O__) {
 #ifdef ALLOWS_INVALID_PTHREAD
-    __libc_format_log(ANDROID_LOG_WARN, "libc", "invalid pthread_t %p passed to libc", thread);
+      async_safe_format_log(ANDROID_LOG_WARN, "libc", "invalid pthread_t %p passed to libc", thread);
 #else
     if (thread == nullptr) {
       // This seems to be a common mistake, and it's relatively harmless because
-- 
2.17.1

